### PR TITLE
Version bump to 1.13.2.0

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: kibana-reports/server/routes/utils/converters/backendToUi.ts [hunk 1]
```diff
@@ -222,12 +222,15 @@ const getUiQueryUrl = (
   const timeFrom = moment(beginTimeMs).toISOString();
   const timeTo = moment(endTimeMs).toISOString();
   let queryUrl = `${baseUrl}?_g=(time:(from:'${timeFrom}',to:'${timeTo}'))`;
-  if (tenant) {
-    if (tenant === '__user__') {
+  if (tenant !== undefined) {
```
Comment (davidcui1225) [inline kibana-reports/server/routes/utils/converters/backendToUi.ts:225]: What's this change? I don't think we planned to add this to the 1.13.2 release?

**Review by davidcui1225 — APPROVED** [review]

File: kibana-reports/server/routes/utils/converters/backendToUi.ts (hunk 1 above)
Comment (joshuali925) [reply kibana-reports/server/routes/utils/converters/backendToUi.ts:225]: this is from https://github.com/opendistro-for-elasticsearch/kibana-reports/pull/324 which zhongnan backported to 1.13 branch. this seems like a github issue to me, i don't think it should show up in diff..

**Review by joshuali925 — APPROVED** [review]

File: kibana-reports/server/routes/utils/converters/backendToUi.ts (hunk 1 above)
Comment (zhongnansu) [reply kibana-reports/server/routes/utils/converters/backendToUi.ts:225]: cherry-pick creates new commit id, and github diff I guess is not comparing the actual diff, instead if somehows uses commid id to compare, just a guess

Comment (joshuali925) [reply kibana-reports/server/routes/utils/converters/backendToUi.ts:225]: git should be able to figure out commits that have same content with different hash, I think github uses 3 dot diff `git diff ours...theirs` to preview merge, but this result is weird. I merged locally and only version bumps are in the diff so shouldn't affect us

**Pull request merged**